src/django_pg_migration_tools/operations.py

```diff
@@ -49,10 +49,11 @@ class due to limitations of Django's AddIndexConcurrently operation.
 
     DROP_INDEX_QUERY = 'DROP INDEX CONCURRENTLY IF EXISTS "{}";'
 
-    def __init__(self, model_name: str, index: models.Index) -> None:
+    def __init__(self, model_name: str, index: models.Index, hints: Any = None) -> None:
         self.model_name = model_name
         self.index = index
         self.original_lock_timeout = ""
+        self.hints = hints or {}
```

Comment:
🐼 

```suggestion
        self.hints = {} if hints is None else hints
```

Reply:
Fair enough. The original does what [Django does](https://github.com/django/django/blob/7adb6dd98d50a238f3eca8c15b16b5aec12575fd/django/db/migrations/operations/special.py#L81), though I don't see harm on changing this.

Fixup: 77942bb7ba377deff5954cbb290fde9609108b80